docroot/themes/contrib/civictheme/includes/node.inc

```diff
@@ -107,3 +107,64 @@ function _civictheme_preprocess_page__node(&$variables) {
     $variables['hide_sidebar'] = TRUE;
   }
 }
+
+/**
+ * Pre-process CivicTheme nodes Slider slide view mode.
+ */
+function _civictheme_preprocess_node__civictheme_slider_slide(&$variables) {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $variables['node'];
+
+  $variables['title'] = $node->label();
+
+  // Image.
+  if ($node->hasField('field_c_n_thumbnail') && !$node->get('field_c_n_thumbnail')->isEmpty()) {
```

Comment:
Page has banner images 
Event has featured images

-- Used a common image to display "thumbnail"